api/desecapi/tests/testrrsets.py

```diff
@@ -189,6 +190,107 @@ def test_create_my_rr_sets(self):
                 self.assertStatus(response, status.HTTP_200_OK)
                 self.assertRRSet(response.data, **data)
 
+    def test_bulk_post_my_rr_sets(self):
+        data = []
+        for subname in ['', 'create-my-rr-sets', 'foo.create-my-rr-sets', 'bar.baz.foo.create-my-rr-sets']:
+            data.extend([
+                {'subname': subname, 'records': ['1.2.3.4'], 'ttl': 60, 'type': 'A'},
+                {'subname': subname, 'records': ['desec.io.'], 'ttl': 900, 'type': 'PTR'},
+            ])
+
+        with self.assertPdnsRequests(self.requests_desec_rr_sets_update(name=self.my_empty_domain.name)):
+            response = self.client.bulk_post_rr_sets(domain_name=self.my_empty_domain.name, payload=data)
+            self.assertStatus(response, status.HTTP_201_CREATED)
+
+        response = self.client.get_rr_sets(self.my_empty_domain.name)
+        self.assertStatus(response, status.HTTP_200_OK)
+        self.assertRRSetsCount(response.data, data)
+
+        for value in data:
```

Comment:
move this into a single call, e.g. `self.assertAPIRRSet(domain_name, subname, type_, kwargs)` where `kwargs = value`

Reply:
Similar, we could implement `self.assertDBRRSet` to check directly against the database (see also your TODO)

Reply:
no necessary as of now, we just get all RRsets via API and then use `assertRRSetsCount()`